pkg/cli/start.go

```diff
@@ -24,58 +32,142 @@ var startCmd = &cobra.Command{
 	Start the telemetry server, which will start listening for data on
 	the input device specified via the --source flags
 	`,
-	Example: `  telemetry start --schema=/foo.asciipb [--port=port] [--fake] [--tty=/dev/...] [--db=a.db]`,
-	RunE:    runStart,
+	Example: `
+	telemetry start --schema=/foo.asciipb
+									[--port=port]
+									[--source=(s|f|r)]
```

Comment:
Since the command line arguments are changing, could you update the README (under telemetry-server setup) to use `--source=f` instead of `-f`?